Package.swift

```diff
@@ -14,24 +14,34 @@ let package = Package(
         .library(name: "Amplitude", targets: ["Amplitude"]),
     ],
     targets: [
-        .target(
-            name: "Amplitude",
-            path: "Sources/Amplitude",
-            exclude: ["AppledocSettings.plist"],
+          .target(
+            name: "EventExplorer",
+            path: "Sources/EventExplorer",
             resources: [
                 .process("Resources/AMPBubbleView.xib"),
                 .process("Resources/AMPInfoViewController.xib"),
-                .process("Resources/cancel.png"),
-                .process("Resources/cancel@2x.png"),
-                .process("Resources/cancel@3x.png"),
-                .process("Resources/ComodoRsaDomainValidationCA.der"),
-                .process("Resources/logo-banner.png"),
-                .process("Resources/logo-banner@2x.png"),
-                .process("Resources/logo-banner@3x.png"),
-                .process("Resources/logo-button.png"),
-                .process("Resources/logo-button@2x.png"),
-                .process("Resources/logo-button@3x.png")
+                .process("Resources/Images/cancel.png"),
```

Comment:
When you move these files, make sure the `Target Membership` for them is not changed in xcode. They (EE related) should be only for iOS.

Reply:
this was not changed so we should be good